iree/compiler/Dialect/VM/Conversion/VMToEmitC/DropDisabledFunctions.cpp

```diff
@@ -0,0 +1,54 @@
+// Copyright 2021 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "iree/compiler/Dialect/VM/Conversion/VMToEmitC/DropDisabledFunctions.h"
+
+#include "mlir/Pass/Pass.h"
+
+namespace mlir {
+namespace iree_compiler {
+namespace IREE {
+namespace VM {
+
+class DropDisabledFunctionsPass
```

Comment:
took me a bit to figure out what "disabled" meant - maybe use "excluded"/`emitc.exclude` instead to indicate that you are excluding these functions from emitc output? ("disable" in this context made me think of some attribute of the function being disabled, vs the entire function itself being removed from the module)